error.go

```diff
@@ -2,38 +2,42 @@
 // and efficient error handling.
 package microerror
 
-import (
-	"fmt"
-
-	"github.com/juju/errgo"
-)
-
 var (
-	// MaskAny is a simple error masker. Masked errors act as tracers within the
-	// source code. Inspecting an masked error shows where the error was passed
-	// through within the code base. This is gold for debugging and bug huntin.
-	MaskAny = errgo.MaskFunc(errgo.Any)
+	Default = NewErrgoHandler(ErrgoHandlerConfig{
+		CallDepth: 1,
```

Comment:
You shouldn't use the `Default` directly. It's here only to make it possible for the library user to change the implementation which is used by global functions. I want to test global functions not the `Default` itself as it isn't meant to be used externally. I'll add a comment.

> why does the default config now set the call depth to 1?

Because if you want to use a handler inside a function (I think it would be the default usecase) you want `0` depth. Here `Default` is used inside wrapping `Mask` & `Maskf` functions, that's why depth `1` is desired here.